recursive_art.py

```diff
@@ -80,8 +110,44 @@ def remap_interval(val,
         >>> remap_interval(5, 4, 6, 1, 2)
         1.5
     """
-    # TODO: implement this
-    pass
+
+    # range check
+    if input_interval_start == input_interval_end:
+        print ("Warning: Zero input range")
+        return None
+
+    if output_interval_start == output_interval_end:
+        print ("Warning: Zero output range")
+        return None
+
+    #check reversed input range
+    reverseInput = False
+    oldMin = min( input_interval_start, input_interval_end )
+    # print(oldMin)
```

Comment:
In general, before turning things in it's best to remove these debugging print statements.